layouts/partials/title.html

```diff
@@ -0,0 +1,7 @@
+<title>
+	{{~#if __docTitle}}
```

Comment:
Strips out whitespace to tidy up source code: [Handlebars.js docs: Whitespace Control](http://handlebarsjs.com/expressions.html#whitespace-control)